settings.gradle

```
@@ -18,4 +18,6 @@ include 'services:webservice'
 
 rootProject.name = 'KOMORAN'
 include 'core'
+include 'admin'
+include 'admin'
```

admin 모듈이 한 번만 추가되어도 될 것 같습니다.
```suggestion
include 'admin'
```

---

merge 후에 진행하겠습니다.